src/intro/hello-world.md

```diff
@@ -244,7 +244,7 @@ Rust does not natively support inheritance, but the gdext API emulates it to a c
 
 ### Class declaration
 
-In this example, we declare a class called `Player`, which inherits [`Sprite2D`][api-sprite2d] (a node type):
+In this example, we declare a class called `Player`, which inherits [`Sprite2D`][api-sprite2d] (a node type). Try adding this code into your `lib.rs` file:
```

Comment:
This is incorrect -- classes can be defined anywhere, they don't need to be in `lib.rs`. 

See also `dodge-the-creeps` example. You may have encountered a different problem in your own code.